tests/components/zwave_js/test_button.py

```diff
@@ -39,17 +37,6 @@ async def test_ping_entity(
 
     client.async_send_command.reset_mock()
 
-    await hass.services.async_call(
```

Comment:
We can still check this with `pytest.raises(ValueError)` I think.

Reply:
That didn't work, I think it's because the callback occurs in a separate task